content/soarian/dstu2/general-clinical/allergy-intolerance.md

```diff
@@ -0,0 +1,68 @@
+---
+title: AllergyIntolerance | Soarian DSTU 2 API
+---
+ 
+# AllergyIntolerance  
+
+* TOC
+{:toc}
+
+## Overview
+
+For the AllergyIntolerance resource query, the API returns the current state of the patient's allergy list as charted in Soarian.   The allergy list represents a clinical assessment of a patient's allergy or intolerance when exposed to a specific substance or class of substance.  
+
+Soarian's list supports medication, food, environmental, and other allergen categories.  Allergy and Intolerance are not differentiated.  Charted assertions of no allergy is different than an empty list; the response will include any charted assertions such as NKA (no known allergies), NKDA (no known drug allergy), NKFA (no known food allergy), No Latex Allergy, and/or No IV contrast Allergy.  A consolidated list of allergens, reactions, and severities that providers have charted in the Soarian Allergy List.  Only allergies in the active state are returned.
```

Comment:
"A consolidated list of allergens, reactions, and severities that providers have charted in the Soarian Allergy List."
Seems like an incomplete sentence